bazel/cargo/remote/BUILD.ahash-0.8.3.bazel

```diff
@@ -11,15 +11,15 @@ load("@rules_rust//rust:defs.bzl", "rust_library")
 
 package(default_visibility = ["//visibility:public"])
 
-# licenses([
-#     "TODO",  # MIT OR Apache-2.0
-# ])
-
 rust_library(
     name = "ahash",
-    srcs = glob(["**/*.rs"]),
+    srcs = glob(
+        include = ["**/*.rs"],
+        allow_empty = True,
```

Comment:
Mostly just curious: why is `allow_empty = True` now needed in numerous places? I thought that is the default setting for the glob() function.

Reply:
No idea, this is auto-generated by `//bazel/cargo:crates_vendor`.

Reply:
Got it, thanks.